wgsl/index.bs

```diff
@@ -14352,6 +14356,13 @@ do not correspond directly to types in WGSL.
 This enables a program to write many densely packed values to memory, which can
 reduce a shader's memory bandwidth demand.
 
+Each builtin applies the *inverse* of a [=channel transfer function=] to several input values, then combines
+their results into a single output value. Teh resul
+
+Note: For packing unorm values, the normalized floating point values are in the interval [0,1].
+
+Note: For packing snorm values, the normalized floating point values are in the interval [-1,1].
```

Comment:
For consistency with the previous text, I'd suggest:
```suggestion
Note: For packing unorm values, the normalized floating point values are in the interval [0.0, 1.0].

Note: For packing snorm values, the normalized floating point values are in the interval [-1.0, 1.0].
```

Reply:
took this one.